SpatialGDK/Documentation/content/singleton-actors.md

```diff
@@ -1,55 +1,70 @@
 <%(TOC)%>
 # Singleton Actors
 
-Singleton Actors allow a single source of truth for both operations and data across a multiserver simulation. They are server-side authoritative [Unreal Actors](https://docs.unrealengine.com/en-us/Programming/UnrealArchitecture/Actors) that are restricted to one instantiation on SpatialOS. For example, if you are implementing a scoreboard, you'd most likely only want there to be one of them in your world. Ensuring this behavior in a multiserver paradigm requires a few additional steps which can be easily facilitated through the Singleton Actor.
+You can use a Singleton Actor to define a single source of truth for operations or data, or both, across a game world that uses zoning. You can have only one instance of an [entity]({{urlRoot}}/content/glossary#entity) that represents a Singleton Actor per game world.
 
-There are two kinds of Singleton Actors:
+You define a Singleton Actor by tagging an Actor with the `SpatialType=Singleton` class attribute. For example, if you are implementing a scoreboard, you probably want only one scoreboard in your world, so you can tag the scoreboard Actor as a Singleton Actor.
 
-* **Public Singleton Actors** - Singleton Actors which are replicated to [server-workers and client-workers]({{urlRoot}}/content/glossary#workers). [AGameState](https://docs.unrealengine.com/en-US/Gameplay/Framework/GameMode) is a Public Singleton Actor.
-* **Private Singleton Actors** - Singleton Actors which are replicated to [server-workers]({{urlRoot}}/content/glossary#workers), but not accessible to [client-workers]({{urlRoot}}/content/glossary#workers). [AGameMode](https://docs.unrealengine.com/en-US/Gameplay/Framework/GameMode) is a Private Singleton Actor.
+There are two types of Singleton Actors:
```

Comment:
Looking at this again, I think it should be "Actor" (singular). 

It's similar to "There are many types of dog" (you wouldn't say "many types of dogs").

Reply:
That's an interesting point! 

I googled it and found there had been much discussion on that: 
- https://english.stackexchange.com/questions/29843/types-of-followed-by-singular-or-plural
- https://ffeathers.wordpress.com/2013/09/28/do-you-use-singular-or-plural-after-types-of/

In particular, I noticed that "... there's also variation between British and American English. I speak AmE and I don't think I would ever say there are two types of apple, as a BrE speaker in the linked question mentions is perfectly grammatical there. "

Reply:
Oh wow, it looks like this is somewhat controversial and not clear-cut (like so much of English 🙄)

I've started a poll in the #terminology channel, to see if people have strong opinions... If the singular is a very UK-based thing, then we should avoid it, even though the plural sounds super strange to me!

Reply:
I've just noticed from the #terminology channel "There are two Singleton Actor types", we can use that!